src/view/pay-bitcoin-mobile.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { InputField, AmountInputField } from '../component/field';
+import { Header, Title } from '../component/header';
+import { CancelButton, BackButton, GlasButton } from '../component/button';
+import Card from '../component/card';
+import BitcoinIcon from '../asset/icon/bitcoin';
+import { FormStretcher, FormText } from '../component/form';
+import { BalanceLabel, BalanceLabelUnit } from '../component/label';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  description: {
+    maxWidth: 290,
+  },
+  balance: {
+    marginBottom: 10,
+  },
+  unit: {
+    color: color.blackText,
+  },
+  nextBtn: {
+    marginTop: 20,
+    backgroundColor: color.orange,
+  },
+});
+
+const PayBitcoinView = ({ store, nav, payment }) => (
+  <Background color={color.orange}>
+    <Header color={color.orange}>
+      <BackButton onPress={() => nav.goPay()} />
+      <Title title="Bitcoin Payment">
+        <BitcoinIcon height={13.6} width={10.8} />
+      </Title>
+      <CancelButton onPress={() => nav.goHome()} />
+    </Header>
+    <MainContent>
+      <Card>
+        <FormText style={styles.description}>
+          You are about to initiate an on-chain payment. It could take 10
+          minutes or more to confirm.
+        </FormText>
+        <FormStretcher>
+          <BalanceLabel style={styles.balance}>
+            <AmountInputField
+              autoFocus={true}
+              value={store.payment.amount}
+              onChangeText={amount => payment.setAmount({ amount })}
+              onSubmitEditing={() => nav.goPayBitcoinConfirm()}
+            />
+            <BalanceLabelUnit style={styles.unit}>
+              {store.unitFiatLabel}
+            </BalanceLabelUnit>
+          </BalanceLabel>
+          <InputField
```

Comment:
My invoice PR #821 has fixes for these input fields that should close #824. It might make more sense to review that first, solidify the fixes, apply them, and only then merge this view? Then no need for #824 as well.

We could also separate this view into its own PR, since the rest of the views seem (pretty much) mergeable to me.

Reply:
Ok. I'll take a look at that one next.